### ISSUE TITLE
Deleting a folder from the settings/notes folders doesn't delete content

### ISSUE THREAD

**Issue by pcause** [pbek/QOwnNotes#2367]
#### Expected behaviour

I gop to the notes folder part of settings and remove a notes folder.  I expect the folder and all content to be gone not just from QOwnNotes but also from disk.

#### Actual behaviour

The folder and contents are still physically there.  This is different behavior then if i delete a single note as in that case the content is also deleted from the disk.  I'd expect at least to be asked if I also want to delete al;l content in the folder and if I say yes, it is gone.

I suspect this has something to do with the trash implementation.

#### Steps to reproduce

Create a folder in settings/notes folders
make the new folder the current folder
add/import some content
Go back to settings/notes folders and remove the folder

Using Windows 11 and QOwnNotes 22.1.3

**Comment by pbek** [comment]
No, that setting is just a reference to a folder on a disk. The folder on the disk should not be touched just because the reference to it is removed...
